arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1115,91 +1146,88 @@
         },
         "capacity": {
           "format": "int64",
-          "description": "The number of units being provisioned. Range of values [For F1: 1-1, S1: 1-200, S2: 1-200, S3: 1-10]. To go above this range, call support.",
+          "description": "The number of provisioned IoT Hub units. See: https://docs.microsoft.com/azure/azure-subscription-service-limits#iot-hub-limits.",
           "type": "integer"
         }
-      }
+      },
+      "required": [
+        "name",
+        "capacity"
+      ]
     },
     "EventHubProperties": {
-      "description": "The properties of the provisioned event hub used by the Iot Hub.",
+      "description": "The properties of the provisioned Event Hub-compatible endpoint used by the IoT hub.",
       "type": "object",
       "properties": {
         "retentionTimeInDays": {
```

Comment:
Everything is indeed optional. I will add a link to the default values once we have documented that.